pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/fullacmgkehubfeaturemembership/_http.log

```diff
@@ -1267,52 +1207,22 @@ User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
             "gcpServiceAccountEmail": "sa-${uniqueId}@example-project-01.iam.gserviceaccount.com",
             "httpsProxy": "https://github.com",
             "policyDir": "config-sync-quickstart/multirepo/root",
-            "secretType": "none",
-            "syncBranch": "main",
+            "secretType": "main",
             "syncRepo": "https://github.com/GoogleCloudPlatform/anthos-config-management-samples",
             "syncRev": "264b37c21ef77165d8639d77b23d4ebb8fee053c",
             "syncWaitSecs": "30"
           },
           "sourceFormat": "hierarchy"
         },
-        "hierarchyController": {
-          "enableHierarchicalResourceQuota": false,
-          "enablePodTreeLabels": false,
-          "enabled": false
-        },
+        "hierarchyController": {},
```

Comment:
This is because of custom diff, the GCP API treats all false == {}.  Cherry picking https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1959 to make sure they match